spark/src/main/scala/org/apache/spark/sql/delta/ConflictChecker.scala

```diff
@@ -135,15 +135,22 @@ private[delta] class ConflictChecker(
    * the transaction as if it had started while reading the `winningCommitVersion`.
    */
   def checkConflicts(): CurrentTransactionInfo = {
+    // Protocol and metadata checks.
     checkProtocolCompatibility()
     checkNoMetadataUpdates()
-    checkForAddedFilesThatShouldHaveBeenReadByCurrentTxn()
-    checkForDeletedFilesAgainstCurrentTxnReadFiles()
-    checkForDeletedFilesAgainstCurrentTxnDeletedFiles()
+    checkIfDomainMetadataConflict()
     checkForUpdatedApplicationTransactionIdsThatCurrentTxnDependsOn()
```

Comment:
nit: Add a comment for this one, since it's neither a protocol nor metadata check

Reply:
done.